tests/stress/stress_test.py

```diff
@@ -165,14 +165,16 @@ def _test_large_bundle(self):
         large_file = TestFile('large_file', self._args.large_file_size_gb * 1000)
         dependency_uuid = self._run_bundle([self._cl, 'upload', large_file.name()])
         large_file.delete()
-        self._run_bundle(
+        uuid = self._run_bundle(
             [
                 self._cl,
                 'run',
-                'wc -c large_dependency',
                 'large_dependency:{}'.format(dependency_uuid),
+                'wc -c large_dependency',
```

Comment:
just curious, is the order change significant here?

Reply:
I think it should be dependencies first.